udata_hydra/context.py

```diff
@@ -33,11 +34,9 @@ async def pool():
 
 def queue():
     if "queue" not in context:
+        # we dont need a queue will testing, make sure we're not using a real Redis connection
```

Comment:
Well I think the error `None has no attribute enqueue` would be pretty explicit :p This is really a safeguard because testing can get a bit risky on this codebase.